src/clib/include/feldspar_c99.h

```diff
@@ -56,6 +58,8 @@
 #include "ivar.h"
 #include "taskpool.h"
 
+#define Infinity INFINITY
```

Comment:
What outputs the string "Infinity"?

Reply:
The standard Show instance for floating point types. The Feldspar pretty printer uses 'show' to print literals.

Reply:
The RTS parts of Feldspar are not well-integrated so I have shifted things so Feldspar now mostly outputs legal C99 which eliminates most of the need for the RTS. I think it's better to continue eliminating the RTS, so add a special case for infinity to the `CodeGen Constant` instance for Double/Float.

Reply:
That might be preferable.